usage/Using-the-GUI.md

```diff
@@ -1,45 +1,45 @@
 The GUI for chatoverflow is still in development.  
-Thats why whe have the **Better REPL**, an easy-to-use one-page dashboard, where everything can be controlled.
+That's why whe have the **Better REPL**, an easy-to-use one-page dashboard, where everything can be controlled.
 
 ## Starting the Better REPL
 When using the GUI for the first time use the `[Advanced]` run configuration that packages the gui into the chatoverflow jar.  
 For all further runs you may use the `[Simple]` run configuration which is way faster.  
 
-Once the framework is running open http://localhost:2400/, wait for the server status to display `Connected!` and click on `OPEN BETTER REPL`. 
+Once the framework is running open http://localhost:2400/ in a browser and wait for the server status to display `Connected!`. Now click on `OPEN BETTER REPL`. 
 
 ![](/img/usage/login.png)
 
 When running for the first time enter a password that you want to use for securing your credentials and click on `REGISTER`.  
-If you have already registered, use your password to login.  
-If the login was successful the gui will display the authentication key for that session.  
+If you already are registered, use your password to login.  
+If the login was successful the gui will display the authentication key for this session.  
 
 ## Creating a plugin instance
-Before running a plugin you have to create the plugin instance.  
+Before running a plugin you have to create a plugin instance.  
 All available plugins are listed under `Plugin types`.  
-Add more plugins by putting the plugin Jar files into the plugins folder.
+Add more plugins by putting the plugin `.jar` files into the plugins folder. If you are adding new plugins to the folder when the framework is running, you need to restart the framework in order to load the newly added plugins.
```

Comment:
s/when the framework is running/while the framework is running
s/in order to load/in order for it to load